src/Dapper.FSharp/Dapper.FSharp.fsproj

```diff
@@ -25,5 +30,10 @@
     <Compile Include="Builders.fs" />
     <Compile Include="OptionTypes.fs" />
   </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="All"/>
```

Comment:
Can we use normal paket.dependencies + paket.references files for this or is it necessary to have it in fsproj? 🤔